source/connection-string/tests/valid-options.json

```diff
@@ -20,6 +20,17 @@
       "options": {
         "authmechanism": "MONGODB-CR"
       }
+    },
+    {
+      "description": "Missing delimiting slash between hosts and options",
+      "uri": "mongodb://example.com?tls=true",
+      "valid": true,
+      "warning": false,
+      "hosts": null,
```

Comment:
Looks like `hosts` property is different from one yml.

Reply:
Good catch, forgot to regen the json files